src/main/java/com/mineinabyss/bonfire/data/MessageQueue.kt

```diff
@@ -0,0 +1,8 @@
+package com.mineinabyss.bonfire.data
+
+import org.jetbrains.exposed.dao.id.IntIdTable
+
+object MessageQueue : IntIdTable() {
+    val content = text("content")
```

Comment:
While this will work for now, storing queued messages in the database has the downside that we essentially can't retroactively change system messages after they have been stored in the queue. Players who haven't logged in in a while might still get the old message if we decide to change the message at any point. This is part of a larger problem though, with all our plugin messages to the player being hardcoded. I'd like them to eventually be stored somewhere else with unique identifiers to reference them. So ye thats for later, good for now!

Reply:
As per your request, I will ignore this "for now"